src/jdk/src/macosx/native/sun/awt/OSVersion.m

```diff
@@ -1,61 +0,0 @@
-/*
```

Comment:
This file contains isSnowLeopardOrLower(), which is used in AWTView.m in code that shouldn't be removed.